l5kit/run_tests.sh

```diff
@@ -22,7 +22,7 @@ lint() {
 
 isort() {
   echo "import sorting.."
-  ${PYTHON_EXECUTABLE} -m isort l5kit --check-only --recursive
+  ${PYTHON_EXECUTABLE} -m isort l5kit --check-only
```

Comment:
What's the reason to remove the `--recursive`?

Reply:
they drop the args and made it default. It triggers an error in 5.0.3 and a warning in 5.0.4